src/components/atoms/timestamp.astro

```diff
@@ -0,0 +1,15 @@
+---
+import Typography from '@/atoms/typography.astro';
+
+export interface Props extends astroHTML.JSX.HTMLAttributes {
+  startDate: Date;
+  endDate: Date | null;
+}
+const { startDate, endDate, ...props } = Astro.props;
+const getFormattedDate = (date: Date): string =>
+  new Intl.DateTimeFormat('en-US', { month: 'long' }).format(date).concat(' ', date.getFullYear().toString());
+---
+
+<Typography variant="item-subtitle" class:list={[props.class]}>
```

Comment:
Jak mamy tutaj tylko jedną klasę to prościej będzie użyć zwykłego `class` zamiast `class:list`

Reply:
Wiesz co, eslint mi tu wyrzuca: astro/prefer-class-list-directive

Reply:
Ok, to z lintem się nie kłócę 😄